lib/DTLZ/hpo.py

```diff
@@ -0,0 +1,50 @@
+import os
```

Comment:
performing sorting of imports would be nice (proposed in visual studio code if you use this editor).

Reply:
Yes, it is not 100% alphabetical sorting, [see here for example](https://github.com/deephyper/deephyper/blob/master/deephyper/stopper/_lcmodel_stopper.py)

0. alphabetical ordering within each sub-group
1. `import` before `from ... import ...`
2. standard libraries before external dependencies
3. deephyper imports last